adapters/colossus/colossus.go

```diff
@@ -27,25 +28,17 @@ func Builder(bidderName openrtb_ext.BidderName, config config.Adapter) (adapters
 // MakeRequests create bid request for colossus demand
 func (a *ColossusAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
 	var errs []error
-	var err error
-
-	var bidderExt adapters.ExtImpBidder
-	var colossusExt openrtb_ext.ExtImpColossus
 	var adapterRequests []*adapters.RequestData
 
 	reqCopy := *request
 	for _, imp := range request.Imp {
 		reqCopy.Imp = []openrtb2.Imp{imp}
 
-		if err = json.Unmarshal(reqCopy.Imp[0].Ext, &bidderExt); err != nil {
-			return nil, append(errs, err)
-		}
-		if err = json.Unmarshal(bidderExt.Bidder, &colossusExt); err != nil {
-			return nil, append(errs, err)
-		}
-
-		if colossusExt.TagID != "" {
-			reqCopy.Imp[0].TagID = colossusExt.TagID
+		if tagID, err := jsonparser.GetString(reqCopy.Imp[0].Ext, "bidder", "TagID"); err == nil {
+			reqCopy.Imp[0].TagID = tagID
+		} else if err != jsonparser.KeyPathNotFoundError {
+			errs = append(errs, err)
+			continue
```

Comment:
Would it be feasible to update tests to cover this error case?

Reply:
I'm sorry, but I can't think of a test for this piece of code. Perhaps you can suggest something?

Reply:
I think it's alright to leave this out, since other test coverage looks great